src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -19,8 +19,8 @@
 
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.commands.general.ListCommand;
```

Comment:
With the recent change in behaviour of the `ListCommand`, this will certainly make the code more organised and intuitive for ourselves and peer reviewers!